content/doc/tutorials/tutorial-for-installing-jenkins-on-AWS.adoc

```diff
@@ -255,6 +255,8 @@ Warning: Permanently added 'ec2-198-51-100-1.compute1.amazonaws.com' (RSA) to th
 Completing the previous steps enables you to download and install Jenkins on AWS.
 To download and install Jenkins:
 
+NOTE: The following steps are written for *Amazon Linux 2*. If you're using *Amazon Linux 2023*, it's recommended to use `dnf` instead of `yum`. While the `yum` command is still available for compatibility, it's actually a symbolic link to `dnf` and may not support all of its features. For more details, refer to the https://docs.aws.amazon.com/ko_kr/linux/al2023/ug/package-management.html[official AWS documentation].
```

Comment:
```suggestion
NOTE: The following steps are written for *Amazon Linux 2*. If you're using *Amazon Linux 2023*, it's recommended to use `dnf` instead of `yum`. While the `yum` command is still available for compatibility, it's actually a symbolic link to `dnf` and may not support all of its features. For more details, refer to the https://docs.aws.amazon.com/linux/al2023/ug/package-management.html[official AWS documentation].
```